diff -up xerces-c-3.3.0/src/xercesc/util/MsgLoaders/ICU/ICUMsgLoader.cpp.omv~ xerces-c-3.3.0/src/xercesc/util/MsgLoaders/ICU/ICUMsgLoader.cpp
--- xerces-c-3.3.0/src/xercesc/util/MsgLoaders/ICU/ICUMsgLoader.cpp.omv~	2025-03-09 12:25:07.566755166 +0100
+++ xerces-c-3.3.0/src/xercesc/util/MsgLoaders/ICU/ICUMsgLoader.cpp	2025-03-09 12:25:13.891233820 +0100
@@ -51,8 +51,8 @@ XERCES_CPP_NAMESPACE_BEGIN
  *  The application (this *.cpp) references that symbol here, and will pass the data address to ICU, which
  *  will then  be able to fetch resources from the data.
  */
-#define ENTRY_POINT xercesc_messages_3_2_dat
-#define BUNDLE_NAME "xercesc_messages_3_2"
+#define ENTRY_POINT xercesc_messages_3_3_dat
+#define BUNDLE_NAME "xercesc_messages_3_3"
 
 extern "C" void U_IMPORT *ENTRY_POINT;
 
